src/com/pty4j/PtyProcessBuilder.java

```diff
@@ -37,13 +37,23 @@ public class PtyProcessBuilder {
   public PtyProcessBuilder() {
   }
 
+  public PtyProcessBuilder(@NotNull CommandLine commandLine) {
+    myCommand = commandLine;
+  }
+
   public PtyProcessBuilder(@NotNull String[] command) {
-    myCommand = command;
+    myCommand = new CommandLine.CommandLineList(command);
   }
 
   @NotNull
   public PtyProcessBuilder setCommand(@NotNull String[] command) {
-    myCommand = command;
+    myCommand = new CommandLine.CommandLineList(command);
+    return this;
+  }
+
+  @NotNull
+  public PtyProcessBuilder setRawCommandLine(@NotNull String rawCommandLine) {
```

Comment:
Maybe instead of `setCommand(@NotNull String[] command)` and `setRawCommandLine(@NotNull String rawCommandLine)` we can have a single `setCommandLine(@NotNull CommandLine commandLine)`?
Then, the `CommandLine` instance should be created by clients. WDYT?